js/controller.js

```diff
@@ -230,10 +229,15 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.state.screenToShow = CONSTANTS.SCREEN.LOADING_SCREEN;
     },
 
-    onVcVideoElementCreated: function() {
-      this.state.mainVideoElement = $("#" + this.state.elementId + " .video");
+    onVcVideoElementCreated: function(eventname, params) {
+      var element = $("#" + params["domId"]);
       if (Utils.isIE10()) {
-        this.state.mainVideoElement.attr("controls", "controls");
+        element.attr("controls", "controls");
+      }
+
+      if (params["videoId"] === OO.VIDEO.MAIN)
+      {
+        this.state.mainVideoElement = element;
```

Comment:
So, it is guaranteed that arguments for onVcVideoElementCreated are *correct*, right?